Assets/Plugins/Editor/Uplift/UpliftManager.cs

```diff
@@ -724,13 +726,18 @@ public void UpdatePackage(PackageRepo newer, bool updateLockfile, bool updateDep
 					PackageRepo dependencyPR = PackageList.Instance().FindPackageAndRepository(def);
 					if (Upbring.Instance().InstalledPackage.Any(ip => ip.Name == def.Name))
 					{
-						UpdatePackage(dependencyPR, true, false);
+						UpdatePackage(dependencyPR,
```

Comment:
same

Reply:
and wrong value!

Reply:
I mean this is a potential code path:
```
 l97: InstallPackages(targets, false)
l546:   UpdatePackage(pr, false)
l699:     UpdatePackage(pr, false, true)
l728:       UpdatePackage(dependencyPR, updateLockfile: true, updateDependencies: false);
```

Reply:
what about the `updateDependencies` parameter?

Reply:
Since it was not directly related to this PR objective, I focused on lockfile modifications and I didn't want to modify how `updateDependencies` was transmitted. But I can have a look at it if you think it's necessary.

Reply:
No it's fine for this PR.